pkg/executor/adapter.go

```diff
@@ -1990,8 +1998,6 @@ func (a *ExecStmt) observeStmtBeginForTopSQL(ctx context.Context) context.Contex
 		// Always attach the SQL and plan info uses to catch the running SQL when Top SQL is enabled in execution.
 		if stats != nil {
 			stats.OnExecutionBegin(sqlDigestByte, planDigestByte)
-			// This is a special logic prepared for TiKV's SQLExecCount.
-			sc.KvExecCounter = stats.CreateKvExecCounter(sqlDigestByte, planDigestByte)
```

Comment:
why to delete this line?

Reply:
It's not deleted. GitHub didn't fully update the PR Diff.  https://github.com/YangKeao/tidb/blob/adjust-stmtctx-fields-init-order/pkg/executor/adapter.go looks fine.

Reply:
Change base backward and forward also cannot resolve this issue :facepalm: